lib/alpaca_proxy/api.ex

```diff
@@ -28,21 +28,21 @@ defmodule AlpacaProxy.API do
     async_fetch!(
       build_url(config, conn),
       conn.method,
-      Map.to_list(conn.body_params),
+      conn.body_params,
       headers
     )
   end
 
   @spec async_fetch!(String.t(), method :: String.t(), body_params(), headers()) ::
```

Comment:
When doing such change you need to update the `body_params` type as well.

```diff
-  @spec async_fetch!(String.t(), method :: String.t(), body_params(), headers()) ::
+  @spec async_fetch!(String.t(), method :: String.t(), Conn.params(), headers()) ::
```
and remove no longer needed `body_params` type:
```diff
-  @type body_params :: Conn.params() | []
```